src/DateContentRow.js

```diff
@@ -168,7 +171,7 @@ class DateContentRow extends React.Component {
     while (levels.length < minRows ) levels.push([])
 
     return (
-      <div className={className}>
+      <div className={className} style={style || {}}>
```

Comment:
Super minor, but we can put `style: {}` in `defaultProps` so that don't have to do it here.